CMakeLists.txt

```diff
@@ -33,6 +33,10 @@ execute_process(
 )
 endif()
 
+file(READ "${ZCBOR_DIR}/cose.cmake" COSE_CMAKE_TEXT)
```

Comment:
Please add a comment with JIRA issue, so we can remove those hacks in the future.

Reply:
What about JIRA issue #?

Reply:
Added comment with Jira issue.